packages/client/manifest.json

```diff
@@ -998,70 +998,6 @@
       }
     ]
   },
-  "backgroundimage": {
```

Comment:
For backwards compat, we might be better just to keep this around in the manifest but not expose the component in the builder - seems like over 100 are created monthly.

![Screenshot 2023-01-30 at 16 21 58](https://user-images.githubusercontent.com/11256663/215533706-5a1c7264-4b8b-4507-b6d1-89a5f9e9b235.png)

Reply:
Manifest entry restored.